core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.directive.ts

```diff
@@ -109,6 +109,7 @@ angular.module('oppia').directive('topicEditorNavbar', [
                   var successToast = 'Mail Sent.';
                   AlertsService.addSuccessMessage(
                     successToast, 1000);
+                  $rootScope.$apply();
```

Comment:
Add todos to remove these once the directive is migrated to angular

Reply:
Done